channels/channel.be/vier/chn_vier.py

```diff
@@ -596,23 +596,27 @@ def __extract_stream_collection(self, item, video_info):
                 item.isDrmProtected = True
                 item.metaData["drm"] = drm_key
                 return
-
-            streams = stream_collection["streams"]
-            duration = stream_collection["duration"]
-            if duration:
-                item.set_info_label(MediaItem.LabelDuration, duration)
-            for stream in streams:
-                proto = stream["protocol"]
-                stream_url = stream["url"]
-                if proto == "dash":
-                    stream = item.add_stream(stream_url, 1550 if prefer_widevine else 1450)
-                    Mpd.set_input_stream_addon_input(stream)
-                elif proto == "hls":
-                    stream = item.add_stream(stream_url, 1500)
-                    M3u8.set_input_stream_addon_input(stream)
-
-                if "/geo" in stream_url:
-                    item.isGeoLocked = True
+            
```

Comment:
Keep the original code and change:

`streams = stream_collection.get("streams", None) or []`
`duration = stream_collection.get("duration", 0)`